doc/source/io.rst

```diff
@@ -169,6 +169,13 @@ skipfooter : int, default ``0``
   Number of lines at bottom of file to skip (unsupported with engine='c').
 nrows : int, default ``None``
   Number of rows of file to read. Useful for reading pieces of large files.
+low_memory : boolean, default ``True``
+  Internally process the file in chunks, resulting in lower memory use
+  while parsing, but possibly mixed type inference.  To ensure no mixed
```

Comment:
To complete the example, I see:
```python
df=pd.read_csv(StringIO('\n'.join([str(x) for x in range(1000000)] + ['a string'])))
DtypeWarning: Columns (0) have mixed types. Specify dtype option on import or set low_memory=False.

type(df.loc[524287,'0'])
Out[50]: int

type(df.loc[524288,'0'])
Out[51]: str
```
The first part of the csv data was seen as only int, so converted to int,
the second part also had a string, so all entries were kept as string.